### DISCUSSION TITLE
Add functionality to `get_area()` so it supports areas bounded by all axes

### DISCUSSION THREAD

**Discussion by JeroSQ** [ManimCommunity/manim#2811] [Suggestions and Proposals]
## Description of proposed feature
Modify the function so it allows you to get the area of a function bounded by any axis. For example, an integral from y0 to y1 (I don't know if there's a proper name for this). Like such:
![example](https://user-images.githubusercontent.com/68753500/155862707-2eef4756-e9c6-4b79-80eb-f38525b05d38.gif)

## How can the new feature be used?
Would improve the graphing tool and add functionality.

## Additional comments
This may be regarded as too specific. Let me know what you think.

**Comment by JeroSQ** [comment]
One way I can think of doing this is the following (assuming the graph is on quadrants I _or_ IV and the function increases, like the picture above):

Let the wanted area be the region _next to_ the curve bounded by the y axis of the function **f(x)** from **y0** to **y1**, then:

- Create a new function **g(x)** which is constant at `y0` up to the first `x0` which satisfies `f(x0)=y0`. (Constant until it reaches the function).
- After `x0`, the function would be the same as the original function `f(x)`.
- Create another function **h(x)** constant at `y1`. 
- Return `get_area(h(x), x_range=[0, x1], bounded_graph=g(x))` where `x1` satisfies `f(x1)=y1`

This method can be generalized to all functions on any quadrant as well as increasing or decreasing with some extra logic.

**Comment by nikolim** [comment]
Hi @JeroSQ
This sounds like a great feature. 
Could you assign the issue to me?

**Comment by JeroSQ** [comment]
I can't assign issues but feel free to work on it. Devs need to decide if this feature can be implemented into the library or into manim-contrib

**Comment by Darylgolden** [comment]
I don't think this is too specific to be implemented, but it should be implemented by rewriting `get_area` such that it can be used on any axis, rather than implementing a different logic for the x and y axes.
